evm_loader/ERC20/test/test_erc20.py

```diff
@@ -310,6 +310,21 @@ def __send_neon_transaction(self, evm_trx_data, ethereum_transaction, need_stora
         return send_transaction(client, trx, self.solana_wallet)
 
 
+def check_deposit_emulation(sender, contract, trx_data):
+    print('\nCheck deposit emulation:')
+    print('sender:', sender)
+    print('contract:', contract)
+    print('trx_data:', trx_data)
+    cli = neon_cli()
+    cli_result = cli.emulate(evm_loader_id, sender + ' ' + contract + ' ' + trx_data)
+    print('cli_result:', cli_result)
+    emulate_result = json.loads(cli_result)
+    print('emulate_result:', emulate_result)
+    assert (emulate_result["exit_status"] == 'succeed')
+    assert(int(emulate_result['result']) == 1)
```

Comment:
Please add checks for returned accounts info from the emulator.

Reply:
Fixed. Add checks of the accounts info